pkg/framework/aries/framework.go

```diff
@@ -54,27 +56,51 @@ func New(opts ...Option) (*Aries, error) {
 		return nil, fmt.Errorf("default option initialization failed: %w", err)
 	}
 
+	// TODO -https://github.com/hyperledger/aries-framework-go/issues/212 Define clear relationship between framework
```

Comment:
The line width on this comment is a bit long for GitHub review panel.

Reply:
I think it depends on the view - unified or split view. In case of split, the lines wrap and no scrolling is required.